tools/backup/values.yaml

```diff
@@ -1,15 +1,16 @@
 image: gcr.io/neo4j-helm/backup
 imageTag: 4.1.0-1
 neo4jaddr: holder-neo4j.default.svc.cluster.local:6362
-bucket: gs://graph-backup-storage/city-graph/
+bucket: gs://test-neo4j
 database: neo4j,system
-credentials: /auth/credentials.json
-secretName: neo4j-service-key
+cloudProvider: gcp
+secretName: "neo4j-gcp-credentials"
 pageCache: 2G
 heapSize: 2G
 fallbackToFull: "true"
 checkConsistency: "true"
 checkIndexes: "true"
 checkGraph: "true"
 checkLabelScanStore: "true"
-checkPropertyOwners: "false"
\ No newline at end of file
+checkPropertyOwners: "false"
+jobSchedule: "0 */12 * * *"
```

Comment:
I don't feel strongly about this but for a lot of production customers, once every 24 hours would be a better default

Reply:
This value is default. User can be change while creating chart.